configs/components/puppet.rb

```diff
@@ -8,6 +8,16 @@
   if platform.is_windows?
     pkg.build_requires "rubygem-win32-dir"
   end
+  # Used to compile binary translation files
+  if platform.is_osx?
+    pkg.build_requires "gettext"
+  elsif platform.is_windows?
+    pkg.build_requires "pl-gettext-#{platform.architecture}"
+  elsif platform.is_aix?
+    pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/aix/#{platform.os_version}/ppc/pl-gettext-0.19.8-2.aix#{platform.os_version}.ppc.rpm"
```

Comment:
yeah honestly I'm indifferent on this one. I don't know enough about WHY we don't support i18n on AIX (though I believe it has something to do with not being able to detect the locale? /cc @branan) to say how likely it is that we will expand support there any time soon. If it's merely a matter of "We need to figure out how but haven't gotten around to it" then it might be time to make that dive. If it's more "we know exactly what's wrong and have no way forward", then I think you're right and we should drop this.